config.json

```diff
@@ -13,7 +13,7 @@
   "online_editor": {
     "indent_style": "space",
     "indent_size": 4,
-    "highlightjs_language": "TODO: specify highlightjs language"
+    "highlightjs_language": "wasm"
```

Comment:
Good comment! Since I haven't been a maintainer on other tracks before, please do help set these norms here! I have indeed gotten lax with this on other projects, so good chance to improve my habits!